src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/MigrationResource.kt

```diff
@@ -71,4 +73,52 @@ class MigrationResource(
     @Schema(description = "Prison id", required = true) @PathVariable prisonId: String,
   ): Page<FindMigrationActivitiesResponse> =
     migrationService.findMigrationActivities(pageRequest, prisonId)
+
+  @PreAuthorize("hasRole('ROLE_NOMIS_ACTIVITIES')")
+  @GetMapping("/activities/{courseActivityId}/migrate")
```

Comment:
very minor but not sure if the "migrate" in endpoint URL is needed, you are just getting a curse activity that happens to be used for a migration